src/main/java/org/folio/services/voucher/VoucherService.java

```diff
@@ -85,6 +86,7 @@ public Future<Voucher> getVoucherById(String voucherId, RequestContext requestCo
 
   public Future<VoucherCollection> getVouchers(int limit, int offset, String query, RequestContext requestContext) {
     RequestEntry requestEntry = new RequestEntry(VOUCHER_ENDPOINT).withQuery(query).withLimit(limit).withOffset(offset);
+    LoggingHelper.logQuery("getVouchers", requestEntry);
```

Comment:
I am not sure that logging GET requests will help us, basically we need more attention on different if staments in business log to understand in what condition code goes, also we need to make sure that all catch block properly logged

Reply:
All catch logs are properly handled as they
1. Contain proper `log.error` line
or
2. Rethrow some exception
Actually sometimes there are both which mean a duplicated log but no missing logs.